lib/contentful/sys_data.ex

```diff
@@ -3,21 +3,27 @@ defmodule Contentful.SysData do
   The SysData represents internal additional data for Contentful API objects, usually found in the
   "sys" part of the response objects. It's also referred to as "common properties".
 
-  See the [official documentation for more information](https://www.contentful.com/developers/docs/references/content-delivery-api/#/reference/locales).
+  See the [official documentation for more information](https://www.contentful.com/developers/docs/references/content-delivery-api/#/introduction/common-resource-attributes).
   """
-  defstruct [:id, :revision, :version, :created_at, :updated_at, locale: nil, content_type: nil]
+  defstruct [
+    :id,
+    :revision,
```

Comment:
version wasn't a field I saw documented in the list of common attributes for API entities, and that field was never used in the code, so have removed it

Reply:
good to know, apologies I didn't find any documentation on it, but I must have missed it